frontend/src/container/ListOfDashboard/TableComponents/DeleteButton.tsx

```diff
@@ -51,13 +56,22 @@ const mapDispatchToProps = (
 	deleteDashboard: bindActionCreators(DeleteDashboard, dispatch),
 });
 
-type DeleteButtonProps = Data & DispatchProps;
+export type DeleteButtonProps = Data & DispatchProps;
 
 const WrapperDeleteButton = connect(null, mapDispatchToProps)(DeleteButton);
 
 // This is to avoid the type collision
 function Wrapper(props: Data): JSX.Element {
-	const { createdBy, description, id, key, lastUpdatedTime, name, tags } = props;
+	const {
+		createdBy,
+		description,
+		id,
+		key,
+		refetchDashboardList,
+		lastUpdatedTime,
+		name,
```

Comment:
What does the name represent here? Dashboard name?

Reply:
name of the dashboard